src/Symfony/Bundle/DebugBundle/Resources/config/services.xml

```diff
@@ -8,12 +8,13 @@
         <service id="twig.extension.dump" class="Symfony\Bridge\Twig\Extension\DumpExtension" public="false">
             <tag name="twig.extension" />
             <argument type="service" id="var_dumper.cloner" />
+            <argument type="service" id="var_dumper.html_dumper" />
         </service>
 
         <service id="data_collector.dump" class="Symfony\Component\HttpKernel\DataCollector\DumpDataCollector">
             <tag name="data_collector" id="dump" template="@Debug/Profiler/dump.html.twig" priority="240" />
             <argument type="service" id="debug.stopwatch" on-invalid="ignore" />
-            <argument>null</argument><!-- %templating.helper.code.file_link_format% -->
+            <argument>%kernel.file_link_format%</argument>
```

Comment:
I don't like the fact that this is now "global". We don't have that many global `kernel.*` arguments and this one does not feel like it should be one of them.